modules/vault-cluster/main.tf

```diff
@@ -194,15 +194,46 @@ resource "aws_s3_bucket" "vault_storage" {
   }
 }
 
+resource "aws_dynamodb_table" "vault_dynamo" {
+  count          = "${var.enable_dynamo_backend ? 1 : 0}"
+  name           = "${var.dynamo_table_name}"
+  hash_key       = "Path"
+  range_key      = "Key"
+  read_capacity  = "${var.dynamo_read_capacity}"          #Defaults to 5
+  write_capacity = "${var.dynamo_write_capacity}"         #Defaults to 5
+
+  attribute {
+    name = "Path"
+    type = "S"
+  }
+
+  attribute {
+    name = "Key"
+    type = "S"
+  }
+
+  tags {
+    Description = "Used for HA storage with Vault."
```

Comment:
I believe we already take in a list of custom tags; we should use it here.

Reply:
Added Name tag.  Didn't see any other list of tags anywhere.